### ISSUE TITLE
Strange formatting error on escaped square brackets in a kdoc

### ISSUE THREAD

**Issue by ZacSweers** [facebookincubator/ktfmt#324]
The following kdoc yields a nondescript error. It's a valid doc in theory (or at least the IDE has no issue with it), but ktfmt 0.37 reports the pasted error trace on it.

```kotlin
  /**
   * Find more details about results.alternative : https://docs.aws.amazon.com/chime-sdk/latest/dg/process-msgs.html
   *
   * transcription.results\[i].alternatives always contains at least one entry. If it contains more
   * than one entry, the most confident entry is first in the list. In most cases, you can take the
   * first entry in transcription.results\[i].alternatives and ignore the others.
   */
```

error

```
Step 'ktfmt' found problem in 'src/main/kotlin/slack/services/calls/service/core/ChimeMeetingSessionImpl.kt':
Unexpected: KDOC_MARKDOWN_ESCAPED_CHAR
java.lang.RuntimeException: Unexpected: KDOC_MARKDOWN_ESCAPED_CHAR
        at com.facebook.ktfmt.kdoc.KDocFormatter.processToken(KDocFormatter.kt:131)
        at com.facebook.ktfmt.kdoc.KDocFormatter.formatKDoc(KDocFormatter.kt:76)
        at com.facebook.ktfmt.kdoc.KDocCommentsHelper.rewrite(KDocCommentsHelper.kt:41)
        at com.google.googlejavaformat.Doc$Tok.computeBreaks(Doc.java:755)
        at com.google.googlejavaformat.Doc$Level.computeSplit(Doc.java:330)
        at com.google.googlejavaformat.Doc$Level.computeBreakAndSplit(Doc.java:320)
        at com.google.googlejavaformat.Doc$Level.computeBroken(Doc.java:296)
        at com.google.googlejavaformat.Doc$Level.computeBreaks(Doc.java:266)
        at com.google.googlejavaformat.Doc$Level.computeSplit(Doc.java:330)
        at com.google.googlejavaformat.Doc$Level.computeBreakAndSplit(Doc.java:320)
        at com.google.googlejavaformat.Doc$Level.computeBroken(Doc.java:296)
        at com.google.googlejavaformat.Doc$Level.computeBreaks(Doc.java:266)
        at com.google.googlejavaformat.Doc$Level.computeSplit(Doc.java:330)
        at com.google.googlejavaformat.Doc$Level.computeBreakAndSplit(Doc.java:320)
        at com.google.googlejavaformat.Doc$Level.computeBroken(Doc.java:296)
        at com.google.googlejavaformat.Doc$Level.computeBreaks(Doc.java:266)
        at com.facebook.ktfmt.format.Formatter.prettyPrint(Formatter.kt:110)
        at com.facebook.ktfmt.format.Formatter.format(Formatter.kt:85)

```

**Comment by ZacSweers** [comment]
Possibly related – https://youtrack.jetbrains.com/issue/KT-24840

**Comment by cgrushko** [comment]
Yeah, this looks like it's coming from the kotlin compiler. We should graceful degrade though, maybe skip formatting that kdoc.

**Comment by davidtorosyan** [comment]
Yup, this is a duplicate of #329 and is closed by 5f7b0632.
